deepinpy/recons/recon.py

```diff
@@ -18,10 +18,20 @@
 from torch.optim import lr_scheduler 
 
 @torch.jit.script
-def calc_nrmse(gt, pred):
+def calc_nrmse_real(gt, pred):
     resid = pred - gt
     return (torch.real(opt.zdot_single_batch(resid)) / torch.real(opt.zdot_single_batch(gt))).sqrt().mean()
 
+def calc_nrmse_abs(gt, pred):
```

Comment:
I understand your intention, but this is not the same as the NRMSE of the absolute value between x and y. Perhaps you should give this function a different name to reflect that.

Reply:
In addition, I believe if the hyperparam `abs_loss` is set, then this will already calculate the NRMSE of the absolute value. More generally, it will calculate the loss w.r.t. to the absolute value of `gt` and the absolute value of `pred`

Reply:
I don't think `abs_loss` currently has any effect on the NRMSE, but I can just use that logic to simply things.

Reply:
Sounds good. I understand this is to log statistics and not for actual loss functions. My issue is that you are naming it `nrmse_abs` but your implementation is normalizing by the max/min, which is not the definition of NRMSE.

Reply:
I'm pretty sure nrmse by definition is normalized (i.e. normalized root mean squared error), without normalization it would simply be the rmse. Sorry for the poor naming, but I can create a toggle between rmse and nrmse.